### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by eddsaura**
## Update packages with alerts

To-do:
- [x] Axios
- [ ] utils-extend
- [ ] path-parse
- [ ] nth-check
- [ ] ansi-regex

Used the packagejson resolutions to fix whatever we could.

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "@types/react": "17.0.5",
     "@types/styled-components": "5.1.9",
     "@types/yup": "0.29.11",
-    "axios": "0.21.1",
+    "axios": "^0.24.0",
```
Comment (jordisala1991) [inline frontend/package.json:24]:
```suggestion
    "axios": "0.24.0",
```

We are not using ^ in versions (we are fixing the version on the package.json. This might be discussed and changed in another PR, but I would recommend not doing it here, because that would make the package.json inconsistent.

**Review by italodr — APPROVED** [review]
